it_zwierzak/homepage/forms.py

```diff
@@ -0,0 +1,14 @@
+from django.forms import ModelForm
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth.models import User
+from django import forms
+
+
+class CreateUserForm(UserCreationForm):
+    class Meta:
+        model = User
+        fields = {'username', 'email', 'password1', 'password2'}
+
+
+
+
```

Comment:
zgodnie z pep8 po ostatniej napisanej linijce powinien być enter
```suggestion
        fields = {'username', 'email', 'password1', 'password2'}

```